helm_deploy/hmpps-template-typescript/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
delete the `/hmpps-template-typescript` directory

Reply:
done